include/umf/pools/pool_jemalloc.h

```diff
@@ -19,7 +19,8 @@ extern "C" {
 
 /// @brief Configuration of Jemalloc Pool
 typedef struct umf_jemalloc_pool_params_t {
-    /// Set to true if umfMemoryProviderFree() should never be called.
+    /// DEPRECATED and unused, will be removed in the next release,
```

Comment:
Let's not change interfaces without discussion. I am worried that we are changing interfaces carelessly.

Why are you removing this option? Is it a proper fix? Let's discuss first. It is a bad practice that we are changing interfaces without deep analysis. Let's discuss it at the UMF technical meeting.

Reply:
> Let's not change interfaces without discussion. I am worried that we are changing interfaces carelessly.
> 
OK

> Why are you removing this option? 
>

Enabling this option will always cause a memory leak.